app/app/v1alpha/app.proto

```diff
@@ -60,8 +58,8 @@ message App {
   repeated string tags = 6 [(google.api.field_behavior) = OPTIONAL];
   // metadata for the app.
   oneof metadata {
-    // The ai assistant app metadata.
-    AiAssistantAppMetadata ai_assistant_app = 7 [(google.api.field_behavior) = OUTPUT_ONLY];
+    // The AI assistant app metadata.
+    AIAssistantAppMetadata ai_assistant_app = 7 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
@Yougigun this is a breaking change in the `protogen-go` package generated from this definition. If this is in use already, please beware that some entities have updated their names.

Reply:
got it!